packages/storybook-react/src/stories/Paragraph.stories.tsx

```diff
@@ -1,22 +1,18 @@
 import readme from '@nl-design-system-unstable/documentation/componenten/_paragraph.md?raw';
 import { Meta, StoryObj } from '@storybook/react';
-import { Paragraph } from '@utrecht/component-library-react/dist/css-module/index';
+import { Paragraph } from '@utrecht/component-library-react';
```

Comment:
De css-module variant is nodig omdat anders de CSS niet werkt. Als de CSS wel werkt, dan weet ik even niet waarom 🤔